recipes/uvw/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "uvw" %}
+{% set uvw_version = "2.12.1" %}
```

Comment:
I am not entirely sure how our automatic updates work and whether they might get confused by this naming. I think it's more standard to call this `version`.

Reply:
OK, changed